### ISSUE TITLE
[FEATURE] add email account delete confirmation message

### ISSUE THREAD

**Issue by doonfrs** [litespeedtech/cyberpanel#1028]
I hope you have a nice day.
I was trying to remove an account and found that the interface does not ask for confirmation, which is unsafe.
A default javascript confirm message would be enough.

![image](https://user-images.githubusercontent.com/4661009/217657758-de281e10-fa48-4874-8092-1b8f91adadd6.png)

**Comment by aeroboy86** [comment]
@doonfrs - I still cannot believe this has not been implemented. Without a confirmation, you can literally wipe out someones email account with an accidental attempt to change a password

**Comment by doonfrs** [comment]
@aeroboy86 Yes, it is too risky. I created the PR for it, I hope they approve it.

https://github.com/usmannasir/cyberpanel/pull/1428

![Image](https://github.com/user-attachments/assets/78d8b073-fbec-4a7c-8ed4-bbd493ef66db)

**Comment by aeroboy86** [comment]
@usmannasir - could you please incorporate this PR, it is way to risky to not present a confirmation message when deleting an email account. Considering the button is right beside the change password button.

**Comment by usmannasir** [comment]
fixed as per new design.
